cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/flow/BasicFlowletContext.java

```diff
@@ -125,7 +124,8 @@ public int getInstanceId() {
   }
 
   public LoggingContext getLoggingContext() {
-    return new FlowletLoggingContext(getNamespaceId(), getApplicationId(), getFlowId(), getFlowletId());
+    return new FlowletLoggingContext(getNamespaceId(), getApplicationId(), getFlowId(), getFlowletId(),
+                                     getRunId().getId(), String.valueOf(getInstanceId()));
```

Comment:
question : is it better to have ``runId`` before ``flowletId`` ?.  as all the flowlets of a flow have the same runId.